brain/circuit.cpp

```diff
@@ -288,18 +288,32 @@ class MVD3 : public Circuit::Impl
     {
         Vector3fs results( gids.size( ));
         const ::MVD3::Range& range = getRange( gids );
-        const ::MVD3::Positions& positions = _circuit.getPositions( range );
-        assign( range, gids, positions, results, toVector3f );
-        return results;
+        try
+        {
+            const ::MVD3::Positions& positions = _circuit.getPositions( range );
+            assign( range, gids, positions, results, toVector3f );
+            return results;
+        }
+        catch( const HighFive::Exception& )
+        {
+            LBTHROW( std::runtime_error( "Error in getPositions()" ));
```

Comment:
The error message is not very specific. Does HighFive provide any error message in the exception?